server/integration/infinispan/src/main/resources/subsystem-templates/infinispan-core.xml

```diff
@@ -122,6 +129,32 @@
          </cache-container>
       </replacement>
    </supplement>
+   <supplement name="counters">
```

Comment:
This supplement doesn't make sense and is what is causing the wildfly-module tests to fail. No transport is defined, but counters are clustered which results in [this error](https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/factories/RpcManagerFactory.java#L24) being thrown on server startup. AFAICT, this supplement isn't used anywhere either

Reply:
Ok, I'll remove it @ryanemerson

Reply:
Actually, the server throws ^ error whenever you run it standalone in `server/integration/build/target/...` so this isn't just a problem with `wildfly-modules`